db-drda/src/main/java/com/splicemachine/db/impl/drda/NetworkServerControlImpl.java

```diff
@@ -3973,6 +3972,10 @@ void addSession(Socket clientSocket) throws Exception {
                 if ((maxThreads == 0) || (threadList.size() < maxThreads)) {
                     thread = new DRDAConnThread(session, this, getTimeSlice(),
                                                 getLogConnections());
+                    thread.setUncaughtExceptionHandler((t, e) -> {
+                        consoleMessage("tracing unhandled exception originating from thread: '" + t.getName() + "'", true);
```

Comment:
`LOG.debug(message, error)`

Reply:
I would make it part of this [JIRA](https://splicemachine.atlassian.net/browse/DB-10893).